800.renames-and-merges/j.yaml

```diff
@@ -38,6 +38,12 @@
 - { setname: jetty,                    name: [jetty-minimal, jetty-websocket], addflavor: true }
 - { setname: jfrog-cli,                name: jfrog }
 - { setname: jira-cli,                 name: [jira-cli-go, "python:jira-cli"] }
+- { setname: jitsi,                    name: "jitsi-nightly" }
+- { setname: jitsi-meet,               name: "jitsi-meet-nightly" }
+- { setname: jitsi-meet-electron,      name: "datcom-jitsi-meet-electron" }
```

Comment:
This doesn't look correct as it's a fork

Reply:
Ah, your correct. Sorry about this one.